packages/js/product-editor/src/blocks/product-fields/description/components/full-editor-toolbar-button.tsx

```diff
@@ -14,6 +14,26 @@ import { parse, rawHandler } from '@wordpress/blocks';
  */
 import { store } from '../../../../store/product-editor-ui';
 import { getContentFromFreeform } from '../edit';
+import { getGutenbergVersion } from '../../../../utils/get-gutenberg-version';
+
+// There is a bug in Gutenberg 17.9 that causes a crash in the full editor.
+// This should be fixed in Gutenberg 18.0 (see https://github.com/WordPress/gutenberg/pull/59800).
+// Once we only support Gutenberg 18.0 and above, we can remove this check.
```

Comment:
Next time probably you could use [@see jsdoc tag](https://jsdoc.app/tags-see)
Also, I guess it's better using `/**` and `*/` when documenting a function. Something like the following:

```suggestion
/*
 * There is a bug in Gutenberg 17.9 that causes a crash in the full editor.
 * Once we only support Gutenberg 18.0 and above, we can remove this check.
 * This should be fixed in Gutenberg 18.0
 * @see {@link https://github.com/WordPress/gutenberg/pull/59800}
 */
```

Clearly not super relevant here.